Documentation/artifact-format-v3.md

```diff
@@ -270,19 +270,16 @@ As an opposite to the list of global `artifact_provides` being a part of
 `header-info` file, the `artifact_provides` section in the `type-info` file
 is a set of parameters specific for a given payload type.
 
-The list of currently supported parameters is as follows:
-
-* `rootfs_image_checksum` is the checksum of the image contained within the
-  Artifact
+The `artifact_provides` is a key-value store, where the value is either a list,
+or an array of strings.
```

Comment:
Lol, either I missed this the first time, or it was rewritten. "..either a list, or an array...". Aren't those two the same thing? I think the first one should be string.

Reply:
Lol, probably was.